### PR TITLE
feat: BD Live Website UAT

### PR DESCRIPTION
**Pull request by teamchong**
Closes: #40 

## Checklist
- [X] Ran Linting
- [X] Targeted PR against correct branch.
- [X] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [X] Wrote unit tests.
- [ ] Added an entry to the `CHANGELOG.md` file.
- [X] Re-reviewed `Files changed` in the Github PR explorer.

### CODE REVIEW

File: src/screens/AllNetworks/components/SearchBox/useStyles.ts [hunk 1]
```diff
@@ -51,9 +51,13 @@ const useStyles = () => {
       '& .MuiOutlinedInput-notchedOutline': {
         border: 'none',
       },
+      '& input::placeholder': {
+        color: theme.palette.common.white,
+        opacity: 1,
+      }
     }),
```
Comment (teamchong) [inline src/screens/AllNetworks/components/SearchBox/useStyles.ts:58]: https://forbole.atlassian.net/browse/BDU-613

File: src/components/Layout/useStyles.ts [hunk 2]
```diff
@@ -55,11 +55,12 @@ const useStyles = () => {
         border: `1px solid ${theme.palette.text.primary}`,
         color: theme.palette.text.primary,
         /* Neutral Color/White Transparent 01 */
-        backgroundColor: alpha(theme.palette.common.white, 0.3),
+        backgroundColor: alpha(theme.palette.common.white, 0.1),
         boxShadow: theme.shadows[1],
         borderRadius: '50%',
         display: 'none',
         opacity: 0.8,
+        ...theme.mixins.blur02,
```
Comment (teamchong) [inline src/components/Layout/useStyles.ts:63]: https://forbole.atlassian.net/browse/BDU-616

File: src/screens/Home/components/SearchBox/useStyles.ts [hunk 3]
```diff
@@ -51,9 +51,13 @@ const useStyles = () => {
...
         border: 'none',
       },
+      '& input::placeholder': {
+        color: theme.palette.common.white,
+        opacity: 1,
+      }
     }),
     listItem: css({
-      height: `${theme.spacing(7)} !important`,
+      minHeight: `${theme.spacing(7)} !important`,
```
Comment (teamchong) [inline src/screens/Home/components/SearchBox/useStyles.ts:60]: https://forbole.atlassian.net/browse/BDU-620

File: src/screens/Home/components/NetworkBox/useStyles.ts [hunk 4]
```diff
@@ -90,9 +87,6 @@ const useStyles = () => {
         display: 'flex',
         position: 'static',
         gap: theme.spacing(3),
-        '&:hover': {
-          backgroundColor: theme.palette.grey[500],
-        },
         [theme.breakpoints.up('md')]: {
```
Comment (teamchong) [inline src/screens/Home/components/NetworkBox/useStyles.ts:90]: https://forbole.atlassian.net/browse/BDU-621

File: src/screens/Home/components/NetworkBox/useStyles.ts [hunk 5]
```diff
@@ -90,9 +87,6 @@ const useStyles = () => {
         display: 'flex',
         position: 'static',
         gap: theme.spacing(3),
-        '&:hover': {
```
Comment (teamchong) [inline src/screens/Home/components/NetworkBox/useStyles.ts:93]: https://forbole.atlassian.net/browse/BDU-621

File: src/screens/Home/components/NetworkBox/useStyles.ts [hunk 6]
```diff
@@ -17,9 +17,6 @@ const useStyles = () => {
       '& .MuiLinearProgress-root': {
         width: '100%',
       },
-      '&:hover': {
```
Comment (teamchong) [inline src/screens/Home/components/NetworkBox/useStyles.ts:20]: https://forbole.atlassian.net/browse/BDU-621

File: src/screens/AllNetworks/components/NetworkBox/useStyles.ts [hunk 7]
```diff
@@ -115,35 +115,51 @@ const useStyles = () => {
       },
       '& .networkbox__dropdown-btn': {
```
Comment (teamchong) [inline src/screens/AllNetworks/components/NetworkBox/useStyles.ts:116]: https://forbole.atlassian.net/browse/BDU-623

File: src/screens/AllNetworks/components/NetworkBox/index.tsx [hunk 8]
```diff
@@ -45,15 +45,16 @@ const NetworkBox: FC<NetworkBoxProps> = ({ network, isOpened, onOpen }) => {
...
           <NetworkIcon networkName={name} width="48" height="48" />
         </Box>
         <Box>
           <Typography variant="h4">{name}</Typography>
-          <Typography variant="body2">
-            <Badge color="success" badgeContent={links?.length}>
-              {chainId}
-            </Badge>
-          </Typography>
+          <Box className="networkbox__subtitlecontainer">
```
Comment (teamchong) [inline src/screens/AllNetworks/components/NetworkBox/index.tsx:53]: https://forbole.atlassian.net/browse/BDU-623

File: src/screens/AllNetworks/components/NetworkMenuLink/useStyles.ts [hunk 9]
```diff
@@ -19,6 +19,7 @@ const useStyles = () => {
       '& .MuiTypography-subtitle1': {
         /* Secondary Color/Royal Blue 01 */
         color: theme.palette.secondary.dark,
+        fontWeight: '100',
```
Comment (teamchong) [inline src/screens/AllNetworks/components/NetworkMenuLink/useStyles.ts:22]: https://forbole.atlassian.net/browse/BDU-624

File: public/locales/en/about.json [hunk 10]
```diff
@@ -1,4 +1,4 @@
...
...per started as a Cosmos community project in 2018. It was designed as an explorer tool for blockchains that are based on Cosmos SDK, providing optimal user experience with near real-time on-chain data. Big Dipper utilizes different endpoints from Tendermint, Cosmos SDK and the Command Line Interface to aggregate data into a single web interface.\n\nIn 2020, Big Dipper begins to build explorer tools for various protocols such as Celo and Flow. It is a new but exciting adventure for Big Dipper. In this process, we have gradually shaped the modular-based developing strategy, through which Big Dipper will have more flexibility to support different  blockchains in the future. We build each feature as a component that can be added or structured according to the needs of a certain blockchain.\n\nThe core developer of Big Dipper is Forbole, an experienced staking-as-a-service provider and long-term contributor in blockchain space. Forbole has a strong background in network infrastructure, software development and UX design, and our goal with Big Dipper is to build an easy-to-use crypto asset management tool for every crypto enthusiast. Your feedback and suggestions are always appreciated!"
```
Comment (teamchong) [inline public/locales/en/about.json:3]: https://forbole.atlassian.net/browse/BDU-645

**Review by rachelhox — APPROVED** [review]

**Review by rachelhox — APPROVED** [review]

**Pull request merged**